Source/GoBot/ViewDatabase.swift

```diff
@@ -621,6 +621,10 @@ class ViewDatabase {
            .join(self.pubs, on: self.pubs[colMessageRef] == self.msgs[colMessageID])
             .where(self.msgs[colAuthorID] == currentUserID)
             .where(self.msgs[colMsgType] == "pub")
+            .join(authors, on: authors[colAuthor] == pubs[colKey])
+            .join(contacts, on: contacts[colContactID] == authors[colID])
+            .filter(contacts[colAuthorID] == currentUserID)
+            .filter(colContactState == 1)
```

Comment:
These new lines filter out pubs you joined in the past but are no longer following.